x-pack/platform/plugins/shared/cases/public/components/all_cases/utils/stringify_url_params.test.tsx

```diff
@@ -90,7 +90,7 @@ describe('stringifyUrlParams', () => {
     };
 
     expect(stringifyUrlParams(urlParams)).toMatchInlineSnapshot(
-      `"cases=(assignees:!(),category:!(),customFields:(my_field:!(foo,bar)),from:now-30d,owner:!(),page:1,perPage:10,reporters:!(),search:'',searchFields:!(title,description,incremental_id.text),severity:!(),sortField:createdAt,sortOrder:desc,status:!(),tags:!(),to:now)"`
+      `"cases=(assignees:!(),category:!(),customFields:(my_field:!(foo,bar)),from:now-30d,owner:!(),page:1,perPage:10,reporters:!(),search:'',searchFields:!(cases.title,cases.description,cases.incremental_id.text,cases.observables.value,cases.customFields.value,cases-comments.comment,cases-comments.alertId,cases-comments.eventId),severity:!(),sortField:createdAt,sortOrder:desc,status:!(),tags:!(),to:now)"`
```

Comment:
Thinking about this, we won't hit it soon, but this is going to get pretty wild the more attachment types we add and are searchable 🤔. There _should_ be a better way of tracking this. Will add a research ticket to our backlog